### PR TITLE
Make receiver ready widget show more information

### PR DESCRIPTION
**Pull request by bngcebetsha**
We make the `receiver` sensor display WARN == "yellow" if status is not nominal

JIRA: [MT-1533](https://skaafrica.atlassian.net/browse/MT-1533)

### CODE REVIEW

File: app/scheduler/subarray-resources/subarray-resources.html [hunk 1]
```diff
@@ -147,8 +147,8 @@
                             </span>
 
                             <span class="icon-button fa fa-square"
-                                ng-class="{'nominal-sensor-list-item': vm.isReceiverReadySensor(resourceAlloc[0]).value === true,
-                                'warn-sensor-list-item': vm.isReceiverReadySensor(resourceAlloc[0]).value === false}"
+                                ng-class="{'nominal-sensor-list-item': vm.isReceiverReadySensor(resourceAlloc[0]).value == true,
+                                'warn-sensor-list-item': vm.isReceiverReadySensor(resourceAlloc[0]).value == false || vm.isReceiverReadySensor(resourceAlloc[0]).status != 'nominal'}"
```
Comment (ajoubertza) [inline app/scheduler/subarray-resources/subarray-resources.html:151]: Do we need to check the status in the nominal colour case as well?

E.g., If `value` is true, but `status` is `'error'`, will this result in
`ng-class="{'nominal-sensor-list-item', 'warn-sensor-list-item'}?  In that case, what colour will the button be?

Also, can we split these very long lines?  They don't fit in the unified diff view on github.

Comment (bngcebetsha) [reply app/scheduler/subarray-resources/subarray-resources.html:151]: if `value = false` OR `status != 'nominal' display yellow, else green

Comment (ajoubertza) [reply app/scheduler/subarray-resources/subarray-resources.html:151]: OK, but does it make sense to list both classes in the case that we know it should be a warning?

Comment (bngcebetsha) [reply app/scheduler/subarray-resources/subarray-resources.html:151]: Yes

`ng-class` is accepting a `dict()-type` with key-value pairs, it only displays the value that valuates to `true` corresponding to a key:
`d = {'nominal-sensor-list-item': true_value,`
`'warn-sensor-list-item': false_value || status != 'nominal'}"`
 Our keys are: 
- `'nominal-sensor-list-item'` and
- `'warn-sensor-list-item'`
We need them both...

Comment (ajoubertza) [reply app/scheduler/subarray-resources/subarray-resources.html:151]: As discussed - we don't really want both of them.  Either nominal or warn, not both.  The colour from the last one "wins", so it does end up looking correct.  I.e., yellow when there is a warning.

**Pull request merged**